scipy/optimize/tests/test_linesearch.py

```diff
@@ -145,7 +145,7 @@ def test_scalar_search_wolfe1(self):
     def test_scalar_search_wolfe2(self):
         for name, phi, derphi, old_phi0 in self.scalar_iter():
             s, phi1, phi0, derphi1 = ls.scalar_search_wolfe2(
-                phi, derphi, phi(0), old_phi0, derphi(0))
+                phi, derphi, phi(0), old_phi0, derphi(0), maxiter=10)
```

Comment:
This test doesn't actually check anything, since we already know that the test works when `maxiter=10`. You should check to see that the test works for a range of values instead of just 10.

Reply:
Done Thank You!